suite/tests/CMakeLists.txt

```diff
@@ -2782,11 +2811,14 @@ if (X86)
   endif()
 endif ()
 
-tobuild_appdll(client.drwrap-test client-interface/drwrap-test.c)
-get_target_path_for_execution(drwrap_libpath client.drwrap-test.appdll "${location_suffix}")
-tobuild_ci(client.drwrap-test client-interface/drwrap-test.c "" "" "${drwrap_libpath}")
-use_DynamoRIO_extension(client.drwrap-test.dll drwrap)
-tochcon(client.drwrap-test.appdll textrel_shlib_t)
+if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
+  tobuild_appdll(client.drwrap-test client-interface/drwrap-test.c)
+  get_target_path_for_execution(drwrap_libpath client.drwrap-test.appdll "${location_suffix}")
```

Comment:
Line too long: please wrap

Reply:
> I don't know if the too-long line you said that is:
> if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
> Or
> get_target_path_for_execution(drwrap_libpath client.drwrap-test.appdll "${location_suffix}")
> I have already moved # TODO i#3544: Port tests to RISC-V 64 to the second line.
> 
> -if (NOT RISCV64) # TODO i#3544: Port tests to RISC-V 64
> +if (NOT RISCV64)
> + # TODO i#3544: Port tests to RISC-V 64
>   tobuild_appdll(client.drwrap-test client-interface/drwrap-test.c)
>   get_target_path_for_execution(drwrap_libpath client.drwrap-test.appdll "${location_suffix}")

The comment is always immediately below the referenced line.  It is the get_target_path... which is over 80 chars.